pow-migration/src/main.rs

```diff
@@ -255,6 +294,8 @@ async fn main() {
                     }
                     None => {
                         candidate_block += block_windows.readiness_window;
+                        // Instruct to migrate the PoW history up until the next candidate block
+                        tx_candidate_block.send(candidate_block).await.unwrap();
```

Comment:
I think it might be better to just log the error and exit instead of unwraping here